CourseUIKitHW16-20/ViewControllers/UserDefaults + isFirstLaunch.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UserDefaults + is.swift
+//  CourseUIKitHW16-20
+//
+//  Created by Денис Курлыков on 23.04.2023.
+//
+
+import Foundation
+
+extension UserDefaults {
+    // check for is first launch - only true on first invocation after app install, false on all further invocations
+    // Note: Store this value in AppDelegate if you have multiple places where you are checking for this flag
+    static func isFirstLaunch() -> Bool {
+        let hasBeenLaunchedBeforeFlag = "hasBeenLaunchedBeforeFlag"
+        let isFirstLaunch = !UserDefaults.standard.bool(forKey: hasBeenLaunchedBeforeFlag)
+        if (isFirstLaunch) {
```

Comment:
```suggestion
        if isFirstLaunch {
```
Когда берешь фулл код откуда то, переделывай, и мы пишем комнтарии на русском) и переделай функцию на компьютер проперти, если у нас функция ничег оне возвращает, но что то отдает, делаем компьютер.